### PR TITLE
Upgrade kind to 0.7.0(K8s v1.17)

### PR DESCRIPTION
**Pull request by fisherxu**
**What type of PR is this?**
> Uncomment only one ` /kind <>` line, hit enter to put that in a new line, and remove leading whitespaces from that line:
>
> /kind api-change
> /kind bug
> /kind cleanup
> /kind design
> /kind documentation

/kind test
> /kind failing-test
> /kind feature

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

File: hack/local-up-kubeedge.sh [hunk 1]
```diff
@@ -141,7 +141,7 @@ build_edgecore
 
 kind_up_cluster
 
-export KUBECONFIG="$(kind get kubeconfig-path ${CLUSTER_CONTEXT})"
```
Comment (fisherxu) [inline hack/local-up-kubeedge.sh]: Now in kind v0.7.0, it places the kubeconfig to `$HOME/.kube` and named it to config.

File: hack/local-up-kubeedge.sh [hunk 2]
```diff
@@ -141,7 +141,7 @@ build_edgecore
...
+export KUBECONFIG=$HOME/.kube/config
```
Comment (kevin-wangzefeng) [inline hack/local-up-kubeedge.sh]: This is the fix
```suggestion
export KUBECONFIG="$(kind ${CLUSTER_CONTEXT}) get kubeconfig-path"
```

Comment (fisherxu) [reply hack/local-up-kubeedge.sh]:
```shell
➜  ~ kind get
Gets one of [clusters, nodes, kubeconfig]

Usage:
  kind get [command]

Available Commands:
  clusters    Lists existing kind clusters by their name
  kubeconfig  Prints cluster kubeconfig
  nodes       Lists existing kind nodes by their name

Flags:
  -h, --help   help for get

Global Flags:
      --loglevel string   DEPRECATED: see -v instead
  -q, --quiet             silence all stderr output
  -v, --verbosity int32   info log verbosity

Use "kind get [command] --help" for more information about a command.
```
Looks like the  `kubeconfig-path` subcomd  has been removed from `kind get`?

Comment (kevin-wangzefeng) [reply hack/local-up-kubeedge.sh]: from kind v0.6.0 release note:
You do not need to and should not do export KUBECONFIG="$(kind get kubeconfig-path)". This command is deprecated and will be removed in a future release.
kind will set your current-context when creating a cluster
You can configure using a different file by either setting the KUBECONFIG environment variable before creating a cluster, or by setting the --kubeconfig flag on cluster creation
https://github.com/kubernetes-sigs/kind/releases/tag/v0.6.0

**Pull request merged**